scripts/bounty-awaiting-approval-slack-ping.js

```diff
@@ -0,0 +1,90 @@
+// Description:
+//   Script that listens for issues with the label 'bounty-awaiting-approval'
+//   and notifies the collaborators on Slack.
+//
+// Dependencies:
+//   github: "^13.1.0"
+//   probot-config: "^0.1.0"
+//   probot-slack-status: "^0.2.2"
+//
+// Author:
+//   Max Tyrrell (ImFeelingDucky/mac/yung_mac)
+
+const defaultConfig = require('../lib/config')
+const Slack = require('probot-slack-status')
+
+module.exports = (robot) => {
+  robot.log('Connected to bounty-awaiting-approval-slack-ping')
+
+  Slack(robot, (slack) => {
+    robot.log.trace('Connected to Slack')
+
+    checkForNewBounties(robot, slack)
+  })
+}
+
+function checkForNewBounties (robot, slackClient) {
```

Comment:
As this function registers on `'issues.labeled'` event, maybe we can rename it to `registerForNewBounties`? Should we unregister from `'issues.labeled'` at some point?

Reply:
Unregistering is not required as per the Probot samples. Agree with the naming.